tests/integration/examples/test_host_groups.py

```diff
@@ -0,0 +1,87 @@
+import asyncio
+
+import pytest
+
+from adcm_aio_client import ADCMSession, Filter
+from adcm_aio_client.config import ParameterHG
+from adcm_aio_client.objects import ActionHostGroup, ConfigHostGroup
+from tests.integration.examples.conftest import CREDENTIALS, REQUEST_KWARGS
+from tests.integration.setup_environment import ADCMContainer
+
+pytestmark = [pytest.mark.asyncio]
+
+
+async def test_config_host_group(adcm: ADCMContainer) -> None:
```

Comment:
Please add a brief description of the actions demonstrated in this test, such as in the `test_bundle` test.

Reply:
added